src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -248,6 +248,26 @@ public Path copyExistingFileToTarget(Path oldFilePath, Path targetFolder) {
         return null;
     }
 
+    /**
+     * Checks whether a path (defined by a URI) contains a certain sub-path (defined by a URI).
+     *
+     * @param path    URI to be checked
+     * @param subPath URI to check
```

Comment:
The Javadoc and method name mention "contains", but it's actually a "starts with" check. Which is correct / expected?

```suggestion
     * @param path    URI to check if it contains the sub-path
     * @param subPath sub-path URI to search for
```

Reply:
Addressed by [4d90f856a385f6787e592fdcdb13dae79facd8d2](https://github.com/ls1intum/Artemis/pull/7170/commits/4d90f856a385f6787e592fdcdb13dae79facd8d2).